vmm/src/serial_buffer.rs

```diff
@@ -12,6 +16,9 @@ pub(crate) struct SerialBuffer {
     head: usize,
     tail: usize,
     out: Box<dyn Write + Send>,
+    buffering: AtomicBool,
```

Comment:
Why is this atomic? You don't put it in an Arc and share between different threads. Can this be touched by different threads.

Reply:
Ah that's a confused leftover from an earlier revision, can be swapped over to a normal bool.